{{cookiecutter.package_name}}/.zenodo.json

```diff
@@ -0,0 +1,14 @@
+{
+    "license": "BSD-3-Clause",
+    "title": "ihmeuw/{{ cookiecutter.package_name }}: Archival release",
+    "upload_type": "software",
+    "creators": [
+//        {
```

Comment:
Intentional breaking of JSON to prevent accidentally publishing without updating the creators array.